.github/workflows/helm-release.yaml

```diff
@@ -13,17 +13,19 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-
       - name: Configure Git
         run: |
           git config user.name "$GITHUB_ACTOR"
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
-
       - name: Install Helm
         uses: azure/setup-helm@v1
         with:
           version: v3.8.1
-
+      - name: Update chart version
+        run: |
+          VERSION=${GITHUB_REF//refs\/tags\/helm-}
+          sed -i -e 's/^appVersion.*/appVersion: '$RELEASE_VERSION'/' charts/opensearch-operator/Chart.yaml
+          sed -i -e 's/tag.*/tag: '$RELEASE_VERSION'/' charts/opensearch-operator/values.yaml
```

Comment:
Those two lines belong in the release.yaml workflow, here in the helm-release.yaml workflow only the `version` field should be updated using `$VERSION` so it picks the semver version from the tag.